python/pyarrow/tests/test_dataset.py

```diff
@@ -3804,6 +3807,8 @@ def _get_compare_pair(data_source, record_batch, file_format, col_id):
     num_of_files_generated, number_of_partitions \
         = _get_compare_pair(data_source_2, record_batch_1, file_format,
                             partition_column_id)
+    print(f"Generated File Count: {num_of_files_generated}")
```

Comment:
Hi, thanks for the PR!
It is the first time I am contributing to the project reviewing some PRs and trying to understand the project a little better. I was taking a look and from what I can see on the rest of the test files there are not many prints on them. Was it for debugging? I would probably remove the print unless there is a reason to have it.
Thanks

Reply:
Yeah and also left the print statement for
Future debugging purpose 
We can remove it too.

Reply:
Removed the logs